libraries/botbuilder-ai/src/qnamaker-utils/httpRequestUtils.ts

```diff
@@ -31,14 +31,14 @@ export class HttpRequestUtils {
      * @param {string} payloadBody Http request body.
      * @param {QnAMakerEndpoint} endpoint QnA Maker endpoint details.
      * @param {number} timeout (Optional)Timeout for http call
-     * @returns {QnAMakerResult} a promise that resolves to the QnAMakerResult
+     * @returns {QnAMakerResults} a promise that resolves to the QnAMakerResults
      */
     public async executeHttpRequest(
         requestUrl: string,
         payloadBody: string,
         endpoint: QnAMakerEndpoint,
         timeout?: number
-    ): Promise<QnAMakerResult> {
+    ): Promise<QnAMakerResults> {
```

Comment:
This should be amended to include `| undefined` since that's a possible return type.